.gitignore

```diff
@@ -7,3 +7,4 @@ coverage
 .settings
 composer.lock
 docs-api
+.idea/
```

Comment:
Please omit this change and add this to a global `.gitignore` https://help.github.com/articles/ignoring-files/#create-a-global-gitignore